app/components/CollectionPreview.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import * as React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import Collection from "models/Collection";
+import CollectionIcon from "components/CollectionIcon";
+import Highlight from "components/Highlight";
+import CollectionMenu from "menus/CollectionMenu";
+
+type Props = {
+  collection: Collection,
+  highlight?: ?string,
+};
+
+function CollectionPreview(props: Props) {
+  const { collection, highlight } = props;
+
+  return (
+    <CollectionLink to={collection.url}>
+      <Heading>
+        <IconWrapper>
+          <CollectionIcon collection={collection} />
+        </IconWrapper>
+        <Title text={collection.name} highlight={highlight} />
```

Comment:
Thoughts on rendering a summary of the collection description to more closely match the document search results?

Reply:
It makes more sense to me to also allow the collection description to be searchable instead of just showing it.
cc @MatheusRV

Reply:
I don't mind including it in this PR and just landing things the "right way" the first time to be honest. 

This does mean that we should really use the `searchVector` that's stored for collections in the same way on the server as we do for documents, this simplifies things a little as all search results can come from the server and be treated the same instead of some being local.

But, I'm not sure of the best way to deal with pagination with this approach… any thoughts?

Reply:
maybe we can search in the collections first and based on the number of results there we can limit the documents, so we never pass a total of X results for instance.
I just realized that this task is not covering the searches made through slack, is it intentional?

Reply:
I like the idea in theory if you have a good strategy but I any approach that involves coordinating two queries does make pagination extremely difficult… :( I'm struggling to come up with a great way to deal with it beyond the first `offset`.

> I just realized that this task is not covering the searches made through slack, is it intentional?

Yea, we're cool to leave Slack out of this for now.